supabase-replicator/supabase/seed.sql

```diff
@@ -702,26 +701,23 @@ $$ language sql;
 create or replace function get_time()
     returns bigint
     language sql
-    immutable parallel safe
+    stable parallel safe
 as $$
 select (extract(epoch from now()) * 1000)::bigint;
 $$;
 
-
 create or replace function search_contracts_by_group_slugs_for_creator(creator_id text,group_slugs text[], lim int, start int)
     returns jsonb[]
     immutable parallel safe
     language sql
 as $$
 select array_agg(data) from (
     select data
-    from contracts,
-         jsonb_array_elements(data -> 'groupSlugs')
-             as elem
-    where elem ?| group_slugs
+    from contracts
+    where data->'groupSlugs' ?| elem
```

Comment:
If you hadn't indexed the groupSlugs prop I would need that `jsonb_array_elements` or is that the same deal as the rest of my mistaken `json_b` calls?

Reply:
The `jsonb_array_elements` is just a weird way of writing this. If you really wanted it to work like that, you would have had to create an index like, `create index ians_weird_index on contracts (jsonb_array_elements(data->'groupSlugs')). I think that would work?